examples_test_mode.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * PHP MMSoap Example code
+ *
+ * Copyright 2016 MessageMedia
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Uncomment the following line for debugging information
+//ini_set('display_errors', true);
+
+require('mmsoap/MMSoap.php');
+
+// get credentials
+require('credentials.php');
+
+// Set up sendMessage parameters
+// http://www.acma.gov.au/Citizen/Consumer-info/All-about-numbers/Special-numbers/fictitious-numbers-for-radio-film-and-television-i-acma
+$recipients = array($recipient);
+// MessageId array matches each recipient. i.e. 61491570156 will match 24.
```

Comment:
the acma test numbers, and it is declared in credentials.php `$recipient = "+61491570156"; `